site/ja/tutorials/images/classification.ipynb

```diff
@@ -0,0 +1,1366 @@
+    "validation_dir = os.path.join(PATH, 'validation')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Utv3nryxVrV0"
+   },
+   "outputs": [],
+   "source": [
+    "train_cats_dir = os.path.join(train_dir, 'cats')  # 学習用の猫画像のディレクトリ\n",
+    "train_dogs_dir = os.path.join(train_dir, 'dogs')  # 学習用の犬画像のディレクトリ\n",
+    "validation_cats_dir = os.path.join(validation_dir, 'cats')  # 検証用の猫画像のディレクトリ\n",
+    "validation_dogs_dir = os.path.join(validation_dir, 'dogs')  # 検証用の犬画像のディレクトリ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ZdrHHTy2VrV3"
+   },
+   "source": [
+    "### データの理解"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "LblUYjl-VrV3"
+   },
+   "source": [
+    "学習および検証ディレクトリの中にある猫と犬の画像の数を見てみましょう:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "vc4u8e9hVrV4"
+   },
+   "outputs": [],
+   "source": [
+    "num_cats_tr = len(os.listdir(train_cats_dir))\n",
+    "num_dogs_tr = len(os.listdir(train_dogs_dir))\n",
+    "\n",
+    "num_cats_val = len(os.listdir(validation_cats_dir))\n",
+    "num_dogs_val = len(os.listdir(validation_dogs_dir))\n",
+    "\n",
+    "total_train = num_cats_tr + num_dogs_tr\n",
+    "total_val = num_cats_val + num_dogs_val"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "g4GGzGt0VrV7"
+   },
+   "outputs": [],
+   "source": [
+    "print('total training cat images:', num_cats_tr)\n",
+    "print('total training dog images:', num_dogs_tr)\n",
+    "\n",
+    "print('total validation cat images:', num_cats_val)\n",
+    "print('total validation dog images:', num_dogs_val)\n",
+    "print(\"--\")\n",
+    "print(\"Total training images:\", total_train)\n",
+    "print(\"Total validation images:\", total_val)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8Lp-0ejxOtP1"
+   },
+   "source": [
+    "便宜上、データセットの前処理およびネットワークの学習中に使用する変数を設定します。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "3NqNselLVrWA"
+   },
+   "outputs": [],
+   "source": [
+    "batch_size = 128\n",
+    "epochs = 15\n",
+    "IMG_HEIGHT = 150\n",
+    "IMG_WIDTH = 150"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "INn-cOn1VrWC"
+   },
+   "source": [
+    "## Data preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5Jfk6aSAVrWD"
+   },
+   "source": [
+    "Format the images into appropriately pre-processed floating point tensors before feeding to the network:\n",
```

Comment:
I think that we need to translate this paragraph, too.

Reply:
Sorry, I realized this is a draft.

Reply:
Thank you for your comments!
I'll update my document refer to them!